Origin: http://bugs.gentoo.org/attachment.cgi?id=272585&action=diff
Description: libnl headers incompatible with gcc -std=gnu99
Ubuntu-Bug: https://launchpad.net/bugs/780044

Index: libnl-1.1/include/netlink/object.h
===================================================================
--- libnl-1.1.orig/include/netlink/object.h	2011-06-16 13:15:28.000000000 -0500
+++ libnl-1.1/include/netlink/object.h	2011-06-16 13:16:22.000000000 -0500
@@ -55,7 +55,7 @@
 /* Access Functions */
 extern int			nl_object_get_refcnt(struct nl_object *);
 extern struct nl_cache *	nl_object_get_cache(struct nl_object *);
-extern inline void *		nl_object_priv(struct nl_object *);
+extern void *			nl_object_priv(struct nl_object *);
 
 #ifdef __cplusplus
 }
Index: libnl-1.1/lib/object.c
===================================================================
--- libnl-1.1.orig/lib/object.c	2011-06-16 13:16:55.000000000 -0500
+++ libnl-1.1/lib/object.c	2011-06-16 13:17:15.000000000 -0500
@@ -376,7 +376,7 @@
 	return obj->ce_cache;
 }
 
-inline void *nl_object_priv(struct nl_object *obj)
+void *nl_object_priv(struct nl_object *obj)
 {
 	return obj;
 }
